### PR TITLE
add suspicious requests tests

### PR DESCRIPTION
**Pull request by christophe-papazian**
## Description

Add 10 test classes for suspicious request blocking for automatic detection of features in the appsec library dashboard.

- 7 test classes for each request address
- 2 test classes for each response address
-  1 test class for combinations and corner cases

All tests are currently empty

## Workflow

1. ⚠️⚠️ Create your PR as draft (we're receiving lot of PR, it saves us lot of time) ⚠️⚠️
2. Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

> **_NOTE:_**  By default in PR only default scenario tests will be launched. Please refer to the [documentation](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2866381467/CI+Workflow+Github+Actions) to run all scenarios in your PR if needed.

Once your PR is reviewed, you can merge it ! :heart:

### CODE REVIEW

File: tests/appsec/test_blocking_addresses.py [hunk 1]
```diff
@@ -132,3 +132,185 @@ def test_response_header(self):
     @missing_feature(reason="No endpoint defined yet")
     def test_response_cookies(self):
         assert False
+
+
+@rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2667021177/Suspicious+requests+blocking")
+@coverage.not_implemented
+@released(cpp="?", dotnet="?", php_appsec="?", python="?", nodejs="?", golang="?", ruby="?")
+class Test_Blocking_request_method:
```
Comment (cbeauchesne) [inline tests/appsec/test_blocking_addresses.py:140]: Just python style (and same for others): 

```suggestion
class Test_BlockingRequestMethod:
```

**Review by cbeauchesne — APPROVED** [review]
